packages/vite-plugin-cloudflare/playground/react-spa/__tests__/assets.spec.ts

```diff
@@ -20,3 +22,22 @@ test("returns the home page for not found routes", async () => {
 	const content = await page.textContent("h1");
 	expect(content).toBe("Vite + React");
 });
+
+test("applies _headers", async ({}) => {
+	const response = await fetch(viteTestUrl);
+	expect(response.headers.get("X-Header")).toBe("Custom-Value!!!");
+});
```

Comment:
Headers are only being applied to HTML files so, while this test passes, there are other tests with non-HTML assets that wouldn't.

Reply:
Left a quick todo test for this